src/swagger.yml

```diff
@@ -4064,6 +4118,52 @@ parameters:
       Examples:
       - [{"property":"id","operator":"in","type":"array","value":["7627150d-36bd-4867-89b2-091800a03d24","58afec5c-ece9-4b66-bc55-00d6dc1b65b8"]}]
       - [{"property":"in_shop","operator":"eq","type":"boolean","value":true}]
+  PublicationBriefingInput:
+    name: PublicationBriefingInput
+    in: body
+    required: true
+    schema:
+      type: object
+      required:
+        - name
+        - week
+        - store
+        - advertised_aricles
+        - languages
+        - templates
```

Comment:
Add `created_by`.

Reply:
Ok, I'll have to change the backend too, to make it mandatory